exotica/include/exotica/Problems/UnconstrainedEndPoseProblem.h

```diff
@@ -56,13 +56,19 @@ class UnconstrainedEndPoseProblem : public PlanningProblem, public Instantiable<
     double getRho(const std::string& task_name);
     Eigen::VectorXd getNominalPose();
     void setNominalPose(Eigen::VectorXdRefConst qNominal_in);
+    virtual void preupdate();
```

Comment:
Perhaps we should add it to the documentation that preupdate needs to be called in the solver implementation and where.

Reply:
Is there a page in the docs where this can go?

Reply:
Not yet but you can create one